### PR TITLE
Add the "scroll duration" to props

### PR DESCRIPTION
**Pull request by anagura**
I want to change the scrolling speed.

### CODE REVIEW

**Comment by anagura** [comment]
@gilbarbara 

codeclimate failed but It ’s not the part that I ’ve changed.

**Comment by anagura** [comment]
@gilbarbara 

would you please review this pull request?
I want to change the scrolling speed, so I added the property "scroll duration".

File: src/components/index.js [hunk 1]
```diff
@@ -367,6 +372,25 @@ class Joyride extends React.Component {
...
+  ) => {
+    return (
+      !disableScrolling &&
+      (index !== 0 || (scrollToFirstStep || lifecycle === LIFECYCLE.TOOLTIP)) &&
+      step.placement !== 'center' &&
+      (!step.isFixed || !hasPosition(target)) && // fixed steps don't need to scroll
+      (prevState.lifecycle !== lifecycle &&
+        [LIFECYCLE.BEACON, LIFECYCLE.TOOLTIP].includes(lifecycle))
+    );
+  };
```
Comment (glemmaPaul) [inline src/components/index.js]: Why would you want to extract this? I might have missed the point here but it looks like it didn't change much what was there beforehand right?

**Comment by glemmaPaul** [comment]
Def a +1

File: src/components/index.js (hunk 1 above)
Comment (anagura) [reply src/components/index.js]: because codeclimate failed this part.

I forgot the error message.but I think codeclimate says expression is too long.

Comment (glemmaPaul) [reply src/components/index.js]: Aah okay I see, sorry on my part I wasn't aware!

**Comment by glemmaPaul** [comment]
sorry to bug you on this @gilbarbara, but any way this feature can be merged?

**Comment by anagura** [comment]
thanks for merging!

**Pull request merged**